src/components/Dashboard/TransactionModal/EditTransaction.jsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import { useState, useEffect } from 'react';
 import { useFormik } from 'formik';
+import { useDispatch, useSelector } from 'react-redux';
 import * as Yup from 'yup';
 import styles from './EditTransaction.module.css';
 import Datetime from 'react-datetime';
 import 'react-datetime/css/react-datetime.css';
-import { useSelector, useDispatch } from 'react-redux';
+import categories from '../../../../server/models/categories.json';
 import { updateIsModalEditTransactionOpen } from '../../../redux/Slices/global/globalSlice';
+import { editTransaction, getOneTransaction } from '../../../redux/Slices/finance/operations';
 
 export const EditTransaction = () => {
-  const [showModal, setShowModal] = useState(false);
-  const [selectedOption, setSelectedOption] = useState('expense');
+  const [selectedOption, setSelectedOption] = useState('Expense');
   const [selectedDate, setSelectedDate] = useState(null);
   const [category, setCategory] = useState('');
+  const [transactionId, setTransactionId] = useState('');
```

Comment:
Dalej state mimo że wszystko jest w reduxie ? 🤔

Reply:
tak, bo nie wiem jak to ktoś podepnie do strony żebym mógł zaciągać id z danej operacji. Musi być w ten sposób bo byśmy nic nie przetestowali. Pozniej do wywalenia